src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml.cs

```diff
@@ -151,17 +143,17 @@ private async void NextButton_Click(object sender, RoutedEventArgs e)
                     if (AzureDevOps.ConnectedToAzureDevOps)
                     {
                         AzureDevOps.Configuration.SavedConnection = FileIssueHelpers.CreateConnectionInfo(serverUri, null, null);
-                        ChangeStates(States.EditingServer);
+                        ChangeStates(ControlState.EditingServer);
                     }
                     else
                     {
                         ToggleLoading(false);
-                        ServerComboBox.Focus();
+                        Dispatcher.Invoke(ServerComboBox.Focus);
                     }
                 }
                 else
                 {
-                    MessageDialog.Show("URL format is not valid. Example URL: https://accountname.visualstudio.com");
+                    Dispatcher.Invoke(() => MessageDialog.Show("URL format is not valid. Example URL: https://accountname.visualstudio.com"));
```

Comment:
>URL [](start = 64, length = 3)

There's an AB bug (1456852) about updating this text. It also should be in a resource string. It could come later, or it could be fixed while you're already visiting this line. Your call.

Reply:
it should be fixed